src/pages/contributors/ilan-rabinovitch.adoc

```diff
@@ -0,0 +1,172 @@
+So if you're wandering around the expo hall, you might see a 12-year-old kid talking to a salesperson at some tech company getting really excited about whatever the tech is.
+Usually, the companies that come to SCaLE are happy just to see that level of enthusiasm.
+However, the really smart ones realize that the dad or the mom standing behind that 12-year-old kid is the VP of engineering at the company they've been trying to sell to for the last however many years, that they couldn't get a call back from.
+Now that the parent has just spent time with their 12-year-old kid, they also got to hear the pitch on whatever thing they're selling.
+So it's not _just_ an altruistic event, it works for everyone all the way around, and I'm glad that we can do good from multiple angles.
+
+== What advice would you give to those in leadership roles in regards to why it's important to support open source? What kind of ways can people support open source, whether it's a project specifically or the community at large?
+
+So I think there are two reasons for folks in leadership roles to support open source.
+One would be that your company is probably built on open source in some way.
+Unfortunately, you might not know all the different open-source pieces that are involved in that, but your bill of materials and your supply chain is full of open source.
+If you don't support it, you're chopping down wood in that forest without necessarily planting the next generation of growth that'll happen there.
+When that security vulnerability shows up and you're questioning why someone didn't fix it, it is because the people or company who would have fixed it were not supported.
+
+Beyond that, it's a great education opportunity for the developers in your organizations.
+There's so much they can learn from the open-source world in terms of development best practices, new techniques, new software, or other things that they could be using to power whatever your next projects are.
+Recruiting is always one of the hardest parts we have as engineering leaders, and if open source is a gateway that helps the next generation learn about these concepts and skills that you need to hire for, continuing to contribute to open source and support open source is a way in which you can, once again using that gardening metaphor, create a bunch of seedlings for the next harvest.
+
+You can support open source in any number of ways, such as letting some of the folks on your team contribute to a project that you're using.
+It could be something like helping to fix bugs, test code, or contribute documentation.
+Alternatively, it could be financial contributions either by donating to a foundation or buying software from the companies that write open-source software.
```

Comment:
```suggestion
Alternatively, it could be financial contributions, either by donating to a foundation, or buying software from the companies that write open-source software.
```